src/devices/wec2103.c

```diff
@@ -0,0 +1,95 @@
+/** @file
+    WEC-2103 temperature/humidity sensor
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+ */
+
+/*
+-X 'n=name,m=OOK_PPM,s=2000,l=4000,g=4400,r=9400'
+
+Similar to prologue, kedsum, esperanza_ews, s3318p
+Only available information for this device: https://fcc.report/FCC-ID/WEC-2103
+
+Example datagram:
+ f2 90          6b5        96       1     8
```

Comment:
Needs to be valid markdown, i.e. plain blocks indented by four spaces, lists with "- " prefix